src/main/resources/assets/carpet/scripts/ai_tracker.sc

```diff
@@ -382,6 +322,55 @@ global_functions = {
    },
 };
 
+global_hostile_to_villager = {
+   'drowned' -> 8,
+   'evoker'-> 12,
+   'husk' -> 8,
+   'illusioner' -> 12,
+   'pillager' -> 15,
+   'ravager' -> 12,
+   'vex' -> 8,
+   'vindicator' -> 10,
+   'zoglin' -> 10,
+   'zombie' -> 8,
+   'zombie_villager' -> 8
+};
+
+
+__config() ->{
+    'commands'->{
+        ''->'_command',
+        'clear'->'clear',
+        'toggle boxes'->_()->global_display_boxes = !global_display_boxes,
+        'toggle <display>'->['__toggle',null],
+        'toggle villager_hostile_detection <hostile>'->_(h)->__toggle('villager_hostile_detection',h),
```

Comment:
I don't see the point of rewriting the commands to keep the flat call structure for the sake of it. In this case clearly villager_ was just a prefix to fix issues with lack of commands customization.
villager should be its own path with options like hostile_detection, golem_spawning or buddy_detection

Reply:
ye i thought abt that tbh...